main.go

```diff
@@ -73,86 +81,99 @@ var (
-		"health-addr",
-		":9440",
-		"The address the health endpoint binds to.",
-	)
-	flag.StringVar(
+
+	fs.StringVar(
 		&managerOpts.CredentialsFile,
 		"credentials-file",
 		"/etc/capv/credentials.yaml",
 		"path to CAPV's credentials file",
 	)
-	flag.BoolVar(
+	fs.BoolVar(
 		&managerOpts.EnableKeepAlive,
 		"enable-keep-alive",
 		defaultEnableKeepAlive,
 		"feature to enable keep alive handler in vsphere sessions. This functionality is enabled by default.")
-	flag.DurationVar(
+	fs.DurationVar(
 		&managerOpts.KeepAliveDuration,
 		"keep-alive-duration",
 		defaultKeepAliveDuration,
 		"idle time interval(minutes) in between send() requests in keepalive handler",
 	)
-	flag.StringVar(
+	fs.StringVar(
 		&managerOpts.NetworkProvider,
 		"network-provider",
 		"",
 		"network provider to be used by Supervisor based clusters.",
 	)
+
+	// Flags common between CAPI and CAPV
+
+	logsv1.AddFlags(logOptions, fs)
+
+	fs.StringVar(&managerOpts.MetricsBindAddress, "metrics-bind-addr", "localhost:8080",
+		"The address the metric endpoint binds to.")
+
+	fs.BoolVar(&managerOpts.LeaderElection, "leader-elect", true,
+		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
+
+	fs.DurationVar(&leaderElectionLeaseDuration, "leader-elect-lease-duration", 15*time.Second,
+		"Interval at which non-leader candidates will wait to force acquire leadership (duration string)")
+
+	fs.DurationVar(&leaderElectionRenewDeadline, "leader-elect-renew-deadline", 10*time.Second,
+		"Duration that the leading controller manager will retry refreshing leadership before giving up (duration string)")
+
+	fs.DurationVar(&leaderElectionRetryPeriod, "leader-elect-retry-period", 2*time.Second,
+		"Duration the LeaderElector clients should wait between tries of actions (duration string)")
+
+	fs.StringVar(&watchNamespace, "namespace", "",
+		"Namespace that the controller watches to reconcile cluster-api objects. If unspecified, the controller watches for cluster-api objects across all namespaces.")
+
+	fs.StringVar(&managerOpts.PprofBindAddress, "profiler-address", defaultProfilerAddr,
+		"Bind address to expose the pprof profiler (e.g. localhost:6060)")
+
+	fs.BoolVar(&enableContentionProfiling, "contention-profiling", false,
+		"Enable block profiling, if profiler-address is set.")
+
+	fs.DurationVar(&syncPeriod, "sync-period", defaultSyncPeriod,
+		"The minimum interval at which watched resources are reconciled (e.g. 15m)")
+
+	fs.Float32Var(&restConfigQPS, "kube-api-qps", 20,
+		"Maximum queries per second from the controller client to the Kubernetes API server. Defaults to 20")
+
+	fs.IntVar(&restConfigBurst, "kube-api-burst", 30,
+		"Maximum number of queries that should be allowed in one burst from the controller client to the Kubernetes API server. Default 30")
+
+	fs.IntVar(&webhookOpts.Port, "webhook-port", defaultWebhookPort,
+		"Webhook Server port (set to 0 to disable)")
```

Comment:
```suggestion
	fs.IntVar(&webhookOpts.Port, "webhook-port", defaultWebhookPort,
		"Webhook Server port")
```

0 just means use the default port (9443)